projects/dev-schematics/package.json

```diff
@@ -15,21 +15,22 @@
   "author": "",
   "license": "MIT",
   "schematics": "./src/collection.json",
+  "peerDependencies": {
+    "@spartacus/schematics": "next"
```

Comment:
Should we define next here? What about final releases?

Reply:
I think the idea here is to always use the latest schematics - whether from npm, or from verdaccio (when testing locally).